.github/renovate.json5

```diff
@@ -54,12 +54,12 @@
       "allowedVersions": "~0.8.0"
     },
     {
-      // Limit Elasticsearch version to latest supported version in Camunda v8.3.
+      // Limit Elasticsearch version to latest supported version in Camunda v8.9.
```

Comment:
That's the Camunda version :grin: 
It should be 8.4 in that case.

Reply:
🤦